File: src/client/src/models/story.ts
export default class Story {
    id: string;
    title: string;
    skills: string;
    postedOn: string;
    content: string;

    constructor(
        id: string,
        title: string,
        skills: string,
        postedOn: string,
        content: string,
    ) {
        this.id = id;
        this.title = title;
        this.skills = skills;
        this.postedOn = postedOn;
        this.content = content;
    }
}

File: src/server/Models/DbModels/Story.cs
using System.Text.Json.Serialization;

namespace server.DbModels.Models;

public class Story
{
    public required string Id { get; set; }
    public required string Title { get; set; }
    public required string Content { get; set; }
    public required DateTime PostedOn { get; set; }
    public string? ThumbnailUrl { get; set; }
}